ramls/claimed-returned-request.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request body schema for claimed returned API",
+  "properties": {
+    "dateTime": {
```

Comment:
I think we could  follow the same convention as for declared lost, and name this `itemClaimedReturnedDateTime`?

What do you think?

Reply:
yes, renamed.